.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
 before_script:
   - npm prune
 script:
-  - npm start validate
+  - npm run localstart validate
```

Comment:
This is temporary. Because `p-s` uses itself, it is also broken by the thing this PR is fixing.